wp-content/themes/sample-theme/theme.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "color": {
+      "custom": false,
+      "defaultPalette": false,
+      "defaultGradients": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "color": "var(--xwp-sample-theme--color--primary)",
```

Comment:
Use `theme.json` to pass the theme CSS data to the block editor.

Don't use `theme.json` to generate any CSS. Its only purpose is to transfer the global style and setting definitions to the editor.